### ISSUE TITLE
Formatting rule set not working on Android

### ISSUE THREAD

**Issue by davidbilik** [arturbosch/detekt#2321]
## Expected Behavior
Formatting rules like `NoConsecutiveBlankLines` would be reported

## Observed Behavior

This rules violations are not reported

## Steps to Reproduce
I've created sample [repo](https://github.com/davidbilik/DetektIssueSample/) with detekt setup and empty Android project with one [file](https://github.com/davidbilik/DetektIssueSample/blob/master/app/src/main/java/cz/ackee/sample/detekt/MainActivity.kt) with invalid formatting.

When I clone this repo and run
```
./gradlew detekt 
```
this is the output
```
Ruleset: empty-blocks - 5min debt
	EmptyFunctionBlock - [X] at ...app/src/main/java/cz/ackee/sample/detekt/MainActivity.kt:20:5
Ruleset: naming - 5min debt
	FunctionNaming - [X] at ...app/src/main/java/cz/ackee/sample/detekt/MainActivity.kt:19:5
```

all rules from `formatting` ruleset are missing eventhough the file is pretty messed up.

## Context

I am trying to detect formatting issues.

## Your Environment

* Version of detekt used: 1.5.0
* Version of Gradle used (if applicable): 1.5.0
* Operating System and version: MacOS Catalina
* Link to your project (if it's a public repository): https://github.com/davidbilik/DetektIssueSample/

**Comment by cortinico** [comment]
Thanks for the detailed issue @davidbilik 
The problem you're having is that you haven't defined the dependency for the formatting plugin set:

You should add this to your app `build.gradle` file:

```
dependencies {
    detektPlugins "io.gitlab.arturbosch.detekt:detekt-formatting:[detekt-version]"
}
```

**Comment by cortinico** [comment]
Further info on this page: https://arturbosch.github.io/detekt/#adding-more-rule-sets

**Comment by davidbilik** [comment]
You are very right! Thanks! :)
